### ISSUE TITLE
Issue with "WebGLRenderingContext.drawElements()": …

### ISSUE THREAD

**Issue by Mortoc** [mdn/content#813]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/drawElements

#### What information was incorrect, unhelpful, or incomplete?
The documentation states: 
```
count
    A GLsizei specifying the number of elements to be rendered.
```
But "elements" isn't clear. If I am trying to draw a single gl.TRIANGLE, is it expecting that argument to be 1 (if elements means triangles) or 3 (if elements means indices)?

#### Specific section or headline?
count

#### What did you expect to see?
More precise wording to understand what the function is expecting

#### Did you test this? If so, how?
I have a bug that's related, but because this is unclear I have to test both possibilities each time while testing for my bug.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/webglrenderingcontext/drawelements`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/drawElements
* GitHub URL: https://github.com/mdn/content/blob/master/files/en-us/web/api/webglrenderingcontext/drawelements/index.html
* Report started: 2020-12-31T23:54:07.085Z

</details>

**Comment by himanshugarg** [comment]
The issue was fixed by @dominiccooney 's pr #7417 . Anything more required here?

**Comment by Elchi3** [comment]
Looks good
